Automattic/woocommerce-payments
# Restore delete icons used on transaction pages

Fixes https://github.com/Automattic/woocommerce-payments/issues/1685

#### Changes proposed in this Pull Request
These files were removed to tradition to loading the icons from react components. But they are still use by the transaction pages loaded by css [here](https://github.com/Automattic/woocommerce-payments/blob/dc53d1a3afa801b7c7ae4aac237dad9f44e46bca/assets/css/admin.css#L80)

In future 

#### Testing instructions

* Open WCPay transaction or transaction list page after placing an order with giropay or sofort. Make sure icon shows.

-------------------
